### ISSUE TITLE
[Bug]: No binary in Github artifacts

### ISSUE THREAD

**Issue by kaushalyap** [maximbaz/wluma#97]
### Steps for reproducing the issue

Please take look at the GH release page

### What is the buggy behavior?

No wulma binary on release page, only a  .asc file

### What is the expected behavior?

Wulma release binary being on the release page

### Logs

```shell
RUST_LOG=debug wluma
```

### Version

Latest

### Environment

```shell
Nothing
```

**Comment by edwardando** [comment]
Agreed, where is it? Otherwise I'd suggest to remove it as a possible install method!

**Comment by maximbaz** [comment]
I don't have the hardware to build it at the moment, would like the CI to build it and auto-attach to releases.

**Comment by max-baz** [comment]
CI will now build binaries and attach to releases automatically 👍
